alcs-api/src/logout/logout.controller.ts

```diff
@@ -1,19 +1,14 @@
-import { Controller, Get, Inject, Res, UseGuards } from '@nestjs/common';
+import { Controller, Get, Inject } from '@nestjs/common';
 import { ApiOAuth2 } from '@nestjs/swagger';
 import * as config from 'config';
-import { RoleGuard } from '../common/authorization/role.guard';
-import { UserRoles } from '../common/authorization/roles.decorator';
 import { CONFIG_TOKEN, IConfig } from '../common/config/config.module';
-import { ANY_AUTH_ROLE } from '../common/enum';
 
 @ApiOAuth2(config.get<string[]>('KEYCLOAK.SCOPES'))
 @Controller('logout')
-@UseGuards(RoleGuard)
 export class LogoutController {
   constructor(@Inject(CONFIG_TOKEN) private config: IConfig) {}
 
   @Get()
-  @UserRoles(...ANY_AUTH_ROLE)
   async logout() {
```

Comment:
jsut curious what happens if not authenticated user tries to log out?

Reply:
Just tested and it behaves just like as if you were logged in and goes to the redirect URI, so basically takes you to the login page for keycloak.